CHANGELOG.md

```diff
@@ -34,6 +35,126 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v1.1.0-rc1
+
+We expect that this release candidate is quite close to the final v1.1.0
+release. However, subsequent breaking API changes are still possible.
+
+This release candidate is suitable for implementors, but we do not recommend
+shipping products based on a release candidate API due to the possibility of
+incompatible changes prior to the final release. The following represents the
+changes since v1.0.0:
+
+## Standard Channel
+
+### GRPCRoute has Graduated to GA 🎉
+
+GRPCRoute has graduated to GA (v1) and is now part of the Standard Channel. If
+you are already using the experimental version GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+Leading Contributor: @gnossen
+
+### Service Mesh Support has Graduated to GA 🎉
+
+The standard for using Gateway API for Mesh has formally graduated to GA (v1) and is now part of the Standard Channel.
+
+Service mesh support in Gateway API allows service mesh users to use the same API to manage ingress traffic and mesh traffic, reusing the same policy and routing interfaces. In Gateway API v1.1, routes (such as HTTPRoute) can now have a `Service` as a `parentRef`, to control how traffic to specific services behave. For more information, read the [service mesh](https://gateway-api.sigs.k8s.io/mesh/) documentation or the see the list of [integrations](https://gateway-api.sigs.k8s.io/implementations/#service-mesh-implementation-status).
+
+Leading Contributors: @howardjohn, @keithmattix, @kflynn
```

Comment:
```suggestion
Leading Contributors: @howardjohn, @keithmattix, @mikemorris, @kflynn
```

Reply:
(FTR I like the text that @howardjohn provided. 🙂)

Reply:
Thanks @kflynn! Do you mind updating your suggestion to be in alphabetical order by GitHub username just for the sake of consistency with the other lists?

Reply:
JK, realized I could change your suggest 🙃, hope this tweak is OK.